src/ansys/mapdl/core/math.py

```diff
@@ -164,7 +164,7 @@ def _status(self):
         """Print out the status of all APDLMath Objects"""
         return self._mapdl.run("*STATUS,MATH", mute=False)
 
-    def vec(self, size=0, dtype=np.double, init=None, name=None):
+    def vec(self, size=0, dtype=np.double, init=None, name=None, asarray=False):
```

Comment:
New option to directly get the numpy array, instead of having to do 2 sequential calls for that